dev/com.ibm.ws.security.oauth/resources/com/ibm/ws/security/oauth20/internal/resources/OAuthMessages.nlsprops

```diff
@@ -113,8 +113,12 @@ OAUTH_LENGTH_TOO_LARGE_AND_CHANGED=CWWKS1422I: The value of {0} in the oauthProv
 OAUTH_LENGTH_TOO_LARGE_AND_CHANGED.explanation=The value in the indicated attribute is too large.
 OAUTH_LENGTH_TOO_LARGE_AND_CHANGED.useraction=Set the indicated attribute to a proper value.  
 
+OAUTH_CLIENT_REGISTRATION_ILLEGAL_CHAR=CWWKS1423E: OAuth client registration or update failed because the client data contains an illegal character [{0}].
```

Comment:
Change
"OAuth client registration or update failed because the client data contains an illegal character [{0}]."
To
"The OAuth client registration or update failed because the client data contains the "{0}" illegal character."